notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<p>It's not very clear how to infer that conclusion from the tool output. Maybe some more explanation of the meaning of the metrics used may help. Providing a list of the main anomalies and their explanation may also be useful. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>